bin/varnishd/cache/cache_vrt_var.c

```diff
@@ -246,9 +246,10 @@ VRT_l_beresp_uncacheable(VRT_CTX, VCL_BOOL a)
 	}
 	oc = ctx->bo->fetch_objcore;
 
-	VSLb(ctx->vsl, SLT_TTL, "VCL %.0f %.0f %.0f %.0f %s",	\
-	    oc->ttl, oc->grace, oc->keep, oc->t_origin,		\
-	    ctx->bo->uncacheable ? "uncacheable" : "cacheable");\
```

Comment:
similar thing here, and I think we would already be doing the right thing, but IMHO we should rather just `return` in L243 (after the `SLT_VCL_Error`)